### PR TITLE
HPCC-31454 Enable Parquet interface in client tools

### PR DESCRIPTION
**Pull request by dcamper**
The Parquet shared library is not available in client tools, but parquet (the file type) can at least be referenced in ECL code when using the new generic disk reader code.

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: plugins/parquet/CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ##############################################################################
 
-#    HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems®.
+#    HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems�.
```
Comment (jackdelv) [inline plugins/parquet/CMakeLists.txt:3]: Is this change correct?

**Review by jackdelv — COMMENTED** [review]
Looks good, I just have one question.

File: plugins/parquet/CMakeLists.txt (hunk 1 above)
Comment (dcamper) [reply plugins/parquet/CMakeLists.txt:3]: I didn't make that change myself, and I assumed that some editor was fixing it.  On my system, it now looks like:

#    HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems®.

That is Unicode character 00AE.

I just compared it with the same character in plugins/kafka/CMakeLists.txt and it is the same (00AE).

I think it's correct, just not rendered correctly here.

Comment (jackdelv) [reply plugins/parquet/CMakeLists.txt:3]: Ok, that makes sense.

Comment (dcamper) [reply plugins/parquet/CMakeLists.txt:3]: I just realized that, in my previous comment, I was comparing characters in a difference git branch.  After switching to the right branch I see that this character is wrong.

I will revert this change in this file.

**Review by jackdelv — APPROVED** [review]
Looks good.

**Review by GordonSmith — APPROVED** [review]
Changes look ok - does this mean that parquet will be recognised in the clienttools, while not being included?

**Comment by dcamper** [comment]
Exactly.  It only affects generic disk read code, and specifically whether a file type of 'parquet' is recognized.  With this PR, that will happen even though the Parquet shared libraries are not linked.

**Comment by dcamper** [comment]
@ghalliday Please merge.

**Pull request merged**